<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(0, 76, 153);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>643</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>643</width>
     <height>360</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>643</width>
     <height>360</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(51, 153, 255);
  color: white; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eight">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seven">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_five">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_four">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_six">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_two">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_one">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_three">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zero">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(204,204,255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(153, 153, 255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_show">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>480</width>
      <height>70</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777176</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  background-color: black;
  color: white; 
 border: 1px solid gray;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>61</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(51, 153, 255);
  color: white; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bracker_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bracket_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(153, 153, 255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0, 76, 153);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(51, 153, 255);
  color: white; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CE">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(153, 153, 255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: black;
  color: white; 
 border: 1px solid gray;
}
</string>
    </property>
    <property name="text">
     <string>X =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_field">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  background-color: black;
  color: white; 
 border: 1px solid gray;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(153, 153, 255);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(204, 229, 255);
     color: black; 
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_x">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод в поле x</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_main_show">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод в основное поле</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
